packages/reassure-cli/src/commands/measure.ts

```diff
@@ -4,29 +4,33 @@ import { spawnSync } from 'child_process';
 import type { CommandModule } from 'yargs';
 import { compare, formatMetadata } from '@callstack/reassure-compare';
 import type { PerformanceMetadata } from '@callstack/reassure-compare/lib/typescript/types';
+import { RESULTS_DIRECTORY, RESULTS_FILE, BASELINE_FILE } from '../constants';
+import { printBye, printError, printHello, printLog, printWarn } from '../utils/printer';
+import type { DefaultOptions } from '../types';
 import { getGitBranch, getGitCommitHash } from './git';
 
-const RESULTS_DIRECTORY = '.reassure';
-const RESULTS_FILE = '.reassure/current.perf';
-const BASELINE_FILE = '.reassure/baseline.perf';
-
 type MeasureOptions = {
   baseline?: boolean;
   compare?: boolean;
   branch?: string;
   commitHash?: string;
-};
+} & DefaultOptions;
 
 export async function run(options: MeasureOptions) {
+  printHello(options.logLevel);
+  const log = (data: string) => printLog(options.logLevel, data);
+  const warn = (data: string) => printWarn(options.logLevel, data);
+  const error = (data: string) => printError(options.logLevel, data);
+
   const measurementType = options.baseline ? 'Baseline' : 'Current';
 
   const metadata: PerformanceMetadata = {
-    branch: options?.branch ?? (await getGitBranch()),
-    commitHash: options?.commitHash ?? (await getGitCommitHash()),
+    branch: options?.branch ?? (await getGitBranch(options.logLevel)),
+    commitHash: options?.commitHash ?? (await getGitCommitHash(options.logLevel)),
   };
 
-  console.log(`\n❇️  Running performance tests:`);
-  console.log(` - ${measurementType}: ${formatMetadata(metadata)}\n`);
+  warn(`❇️  Running performance tests:`);
```

Comment:
Why warn? It seems like this should be regular log.

Reply:
Same as the next comment but in this case I think that you are in fact correct. I will ammend